src/replica/replica_stub.cpp

```diff
@@ -191,6 +191,21 @@ METRIC_DEFINE_counter(server,
                       dsn::metric_unit::kRequests,
                       "The number of busy write requests");
 
+METRIC_DEFINE_gauge_int64(server,
+                          bulk_load_running_count,
+                          dsn::metric_unit::kBulkLoads,
+                          "The number of current running bulk loads");
+
+METRIC_DEFINE_gauge_int64(server,
```

Comment:
For the xxx_max_xxx mastrics, how about use percentile? we can do that in next patches.